raiden/tests/integration/test_pythonapi.py

```diff
@@ -78,8 +89,8 @@ def test_register_token(raiden_network, token_amount, contract_manager, retry_ti
     api1.token_network_register(
         registry_address=registry_address,
         token_address=token_address,
-        channel_participant_deposit_limit=UINT256_MAX,
-        token_network_deposit_limit=UINT256_MAX,
+        channel_participant_deposit_limit=TokenAmount(UINT256_MAX),
```

Comment:
nit: maybe we can create a constant for this.

Reply:
I thought about it but felt that `TokenAmount(UINT256_MAX)` is slightly clearer than `MAX_TOKEN_AMOUNT`, because you know exactly what it does without looking up a variable. But I have no problems adding it.

Reply:
That's true as well, let's leave it like this for now.